.circleci/config.yml

```
@@ -102,10 +150,15 @@ deploy_workflow: &deploy_workflow
     tags:
       only: /v.*/
 
-linux: &linux
-  BUILD_OS: linux
+mac_xcode: &mac_xcode
+  xcode: 12.4.0
```

```suggestion
  xcode: 11.7.0
```

11.7.0 uses MacOS 10.15 (Catalina) which is the oldest version available it looks like. This is currently what we're using for manual builds

https://circleci.com/docs/using-macos/#supported-xcode-versions